layers/meta-testdistro/recipes-devtools/tegra-tools/tegra-sysinstall-tools_1.7.0.bb

```diff
@@ -7,16 +7,29 @@ DEPENDS = "zlib"
 
 COMPATIBLE_MACHINE = "(tegra)"
 
-SRC_URI = "https://github.com/madisongh/tegra-sysinstall/releases/download/v${PV}/tegra-sysinstall-${PV}.tar.gz"
+# file://0001-tools-common.in-set-x-for-debug.patch
+SRC_URI = "https://github.com/madisongh/tegra-sysinstall/releases/download/v${PV}/tegra-sysinstall-${PV}.tar.gz \
+           file://0001-lib-tools-common.in-gdb-tegra-buildinfo.patch \
+           file://0002-lib-tools-common.in-retry-tegra-bootinfo-on-fail.patch \
+           file://0003-lib-tools-common.in-keystoretool-luks-srv-app.patch \
+           file://0004-lib-tools-common.in-align-cryptsetup-args-with-NVidia.patch \
+           file://0005-lib-tools-common.in-ignore-secure-boot-enabled.patch \
+           "
```

Comment:
Could the substantive patches here be reworked as PRs to tegra-sysinstall?

Maybe with a configure-time option to select always-on vs. only with secure boot, and an option to choose which trusted app is being used?

Hopefully the debugging-related patches aren't needed any more. :)

Reply:
https://github.com/OE4T/meta-tegra/pull/1064

Reply:
Add --with-key-retrieval-command configure-time option: https://github.com/madisongh/tegra-sysinstall/pull/6